app/form_objects/group_member_form.rb

```diff
@@ -17,7 +18,10 @@ def save
       return false
     end
 
-    new_membership.save!
+    if new_membership.save
+      send_notification_email
+      true
+    end
```

Comment:
```suggestion
    new_membership.save!
    send_notification_email
    
    true
```

Reply:
yes, that makes sense but the controller relies on the return value:

  @group_member_form = GroupMemberForm.new(group_member_params)
    if @group_member_form.save
      redirect_to group_members_path(@group.external_id)
    else
      render :new, status: :unprocessable_entity
    end